### PR TITLE
Add 'arn' column to aws_account table. Closes #414

### LINKED ISSUE #414: Add 'arn' column to aws_account table
**Is your feature request related to a problem? Please describe.**
I'd like to be able to get an account's ARN without getting it out of the AKAs or composing it myself.

**Describe the solution you'd like**
An `arn` column to to be added

**Describe alternatives you've considered**
Getting it out of the AKAs or composing it myself.

**Additional context**

### PR DESCRIPTION
**Pull request by rajeshbal65**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: aws/table_aws_account.go [hunk 1]
```diff
@@ -180,3 +186,12 @@ func accountDataToAkas(ctx context.Context, d *transform.TransformData) (interfa
 
 	return akas, nil
 }
+
+func accountArn(ctx context.Context, d *transform.TransformData) (interface{}, error) {
```
Comment (Subhajit97) [inline aws/table_aws_account.go:190]:
```suggestion
func accountARN(ctx context.Context, d *transform.TransformData) (interface{}, error) {
```

Comment (Subhajit97) [inline aws/table_aws_account.go:190]: No need to add new function for `arn`. Modify the `akas` function

**Review by Subhajit97 — CHANGES_REQUESTED** [review]
@rajeshbal65 There are some changes required. Please see comments, thanks!

File: aws/table_aws_account.go [hunk 2]
```diff
@@ -28,6 +28,12 @@ func tableAwsAccount(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 				Transform:   transform.FromField("Aliases"),
 			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the Account.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(accountDataToAkas, "ARN"),
```
Comment (Subhajit97) [inline aws/table_aws_account.go:35]: Is there any reason to use `FromP`?
You can update `accountDataToAkas` to return the `arn`, and use `EnsureStringArray` transform to form the `akas`

**Review by Subhajit97 — CHANGES_REQUESTED** [review]
@rajeshbal65 Can Can you please rename `accountDataToAkas` to `accountARN`.

**Review by Subhajit97 — APPROVED** [review]
LGTM

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**